specs/language/expressions.tex

```diff
@@ -230,6 +234,46 @@
 refers to an element index that is out of range of the vector type of the
 postfix expression before the dot.
 
+\Sec{Matrix Swizzle}{Expr.Post.Swizzle.Matrix}
+\begin{grammar}
+  \define{matrix-swizzle-component-sequence}\br
+  matrix-zero-indexed-swizzle-sequence\br
+  matrix-one-indexed-swizzle-sequence\br
```

Comment:
@llvm-beanz  One unique thing I was thinking about indicating is that a matrix  swizzle can not be more than 4 elements. That seems to be implied later when we indicate that a swizzle expression is a vector. However, with long vectors coming in SM 6.9 does this 4 element limitation on matrix swizzles go away? https://hlsl.godbolt.org/z/r9o5PKPWr because I am still seeing it in godbolt: https://hlsl.godbolt.org/z/fdnWqq7cK.